stacks/spica/src/bucket/services/bucket.initializer.ts

```diff
@@ -1,17 +1,32 @@
-import {Injectable} from "@angular/core";
+import {EventEmitter, Injectable, Output} from "@angular/core";
 import {RemoveCategory, RouteCategory, RouteService, Upsert} from "@spica-client/core";
 import {PassportService} from "../../passport";
+import {AddBucketComponent} from "../components/add-bucket/add-bucket.component";
+import {BucketIndexComponent} from "../pages/bucket-index/bucket-index.component";
 import {BucketService} from "./bucket.service";
 
 @Injectable()
 export class BucketInitializer {
+  categoryStorageKey: string = RouteCategory.Content;
+  @Output() onBucketCategoryChange = new EventEmitter();
+  buckets = [];
   constructor(
     private bs: BucketService,
     private routeService: RouteService,
     private passport: PassportService
   ) {
+    this.routeService.patchCategory(RouteCategory.Content, {
+      props: {
+        moreTemplate: BucketIndexComponent,
```

Comment:
I think we should change the name of the BucketIndexComponent to BucketActionsComponent or something like that since it does not index buckets anymore, what do you think?